src/controllers/expenses.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+
+const expensesServise = require('../services/expenses.js');
+const usersServise = require('../services/users.js');
```

Comment:
There is a spelling mistake in the variable name 'usersServise'. It should be 'usersService'. Make sure to rename all instances of this variable throughout the code.